tailwind.config.js

```diff
@@ -9,7 +9,16 @@ module.exports = {
                 sans: ["Barlow", ...sansFonts],
                 semi: ["Barlow Semi Condensed", sansFonts],
             },
-            colors: {},
+            colors: {
+                'yellow-2024' : '#ffb435',
+                'shadeyellow-2024' : '#ff9c44',
+                'blue-2024' : '#d8fcf4',
+                'shadeblue-2024' : '#0bb4a1',
+                'shadeblue2-2024' : '#089080',
```

Comment:
ok, kyk untuk idiomatic tailwind lu instead of bikin `blue-2024`, `shadeblue-2024` sama `shadeblue2-2024` keliatannya lebih idiomatic kalo lu bikin something kyk  `azure-500` `azure-600` `azure-700` misal https://tailwindcss.com/docs/customizing-colors  kyk orang yg baca juga bisa liat kalo itu satu family gitu birunya kyk yang keliatannya shadeblue nya highlight / shadow dari bluenya juga gitu loh.

Reply:
prefer pilih some random color buka wikipedia pick a color yang rasanya close neough dan pendek, dibanding lu call it "yellow-2024" kyk might as well panggil dia kyk "honey" atau something